samples/event_manager_proxy/Kconfig

```diff
@@ -29,7 +29,7 @@ config APP_REMOTE_BOARD
 	string "The name of the CORE to be used by remote image"
 	depends on APP_INCLUDE_REMOTE_IMAGE
 	default DOMAIN_CPUNET_BOARD if (BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP_NS)
-	default "nrf54h20pdk_nrf54h20_cpuppr" if BOARD_NRF54H20PDK_NRF54H20_CPUAPP
+	default "nrf54h20dk_nrf54h20_cpuppr" if BOARD_NRF54H20PDK_NRF54H20_CPUAPP
```

Comment:
Two occurrences of `BOARD_NRF54H20PDK_NRF54H20_CPUAPP` in this file

Reply:
Corrected.